app/scripts/views/sign_up.js

```diff
@@ -314,6 +315,7 @@ function (Cocktail, p, BaseView, FormView, Template, AuthErrors, mailcheck,
 
           if (preVerifyToken && account.get('verified')) {
             self.logViewEvent('preverified.success');
+            self.onSignInSuccess(account);
```

Comment:
The purpose of `onSignInSuccess` here is somewhat opaque, any danger in renaming the function something along the lines of `notifyOtherTabsOfSignIn`?